coding-standards/flutter.md

```diff
@@ -1,9 +1,34 @@
 # Flutter Coding Standards
 
-## Coding Style Guide
-
+## Flutter Coding Style Guidelines
+We adapted standards to design and program Flutter, from high-level architectural principles all the way to indentation rules, from Flutter offical git repo :</br>
 https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
+## Dart Coding Style Guidelines
+Dart Style Guide focuses primarily on Dart-specific conventions: https://dart.dev/effective-dart
 
 ## Standard Enforcing Tools
 
-ESLint - 
+`flutter_lints` `dartfmt`
+## **Set up Flutter Linter and Dart Formatter in Android Studio**
+
+### **Flutter Linter:**
+Add `flutter_lints` to your pubspec.yaml file as a dev dependency under the dev_dependencies section:
+```
+dev_dependencies:
+  flutter_lints: ^1.0.0
+```
+run the following command in the terminal to fetch and install the newly added package:
+```
+flutter pub get
+```
+**Configure flutter_lints**:</br>
+Mac Android Studio: "Preferences/Settings" </br>
+Window Android Studio: go to "Languages & Frameworks" > "Dart" > "Lints". Click on the "Configure" link next to the "Package: flutter_lints" option.
+
+**Enable flutter_lints rules**:
+In the "Select Dart Lints" window, check the box next to "flutter_lints" to enable its lint rules. Click "OK" to apply the changes.
```

Comment:
flutter_lints - comes by default with some lint rules?

We don't need to add any additional configuration or rules on top it?

Reply:
Yes, flutter_lints comes with default recommended rules and as of now I don't think any addition on top of it is required.

Reply:
Is there any link where people can go over that default configuration list and learn about it.
If yes can we add that.